### PR TITLE
feat: add minimal viable test script to produce requests on all versions of the OO

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

This PR adds a very simple script to test OO interfaces by producing requests over all 4 versions of the OO.

### CODE REVIEW

File: packages/scripts/src/testnet/OptimisticOracleRequestSender.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    5
+  );
+  console.log("Sending requests to SkinnyOptimisticOracle", skinnyOptimisticOracle.address);
+  const ooSkinnyTx = await skinnyOptimisticOracle.requestPrice(
+    approvedIdentifier,
+    requestTime,
+    ancillaryData,
+    goerliUsdc,
+    "0",
+    "0",
```
Comment (daywiss) [inline packages/scripts/src/testnet/OptimisticOracleRequestSender.ts:58]: not sure if these are bond values, but would be really useful if we could propose with bonds that we could easily get for dev wallets

**Review by daywiss — APPROVED** [review]
thanks!

**Review by mrice32 — APPROVED** [review]
LGTM! Didn't give a ton of scrutiny, as it's just a testnet script.

If you intend for it to be more readable/modifiable, you might want to hardcode the _decoded_ ancillary data and encode it in the script just so the user knows what to expect and the value is easy to modify, but once again, that may not matter depending on how you intend to use this script.

**Comment by daywiss** [comment]
going to merge this

**Pull request merged**